sysutils/ndcctools/Portfile

```diff
@@ -14,7 +14,7 @@ platforms               darwin
 license                 MIT
 
 maintainers             nd.edu:pivie \
-                        openmaintainer
+                        nd.edu:dthain
```

Comment:
Did you mean to change the preceding line, rather than this line?

Reply:
Ah, I misunderstood open maintainer.  I see now it means "permit minor changes".  Change incoming.